src/framework/Framework_Stability_001.test.js

```diff
@@ -0,0 +1,63 @@
+import {
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+  getControllerPluginData,
+} from '../commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller Robustness test',
+  description: 'Tests start/stop functionality of the controller module and repeats 100 times',
+  teardown: restartFramework,
+  steps: [
+    {
+      description: 'Get Controller data and validate whether it is Object or not',
+      test: getControllerPluginData,
```

Comment:
should be a jsonrpc api request

Reply:
Actual getControllerPluginData method is a jsonrpc request. So this comment is marked as resolved